### PR TITLE
rpc: fix Bloom filter response

### PR DESCRIPTION
**Pull request by thomas-nguy**
Closes: #277

## Description

Fix bloom filter query request.

Height needs to be passed explicitly in the grpc query because tendermint will always uses the current context header.
So it wont be possible to extract the queried 'height' from there

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: ethereum/rpc/backend/backend.go [hunk 1]
```diff
@@ -204,7 +204,7 @@ func (e *EVMBackend) EthBlockFromTendermint(
 		}
 	}
 
-	blockBloomResp, err := e.queryClient.BlockBloom(types.ContextWithHeight(block.Height), &evmtypes.QueryBlockBloomRequest{})
+	blockBloomResp, err := e.queryClient.BlockBloom(types.ContextWithHeight(block.Height), &evmtypes.QueryBlockBloomRequest{Height: block.Height})
```
Comment (thomas-nguy) [inline ethereum/rpc/backend/backend.go:207]: Note :

types.ContextWithHeight(block.Height) => Append height in the header to tell tendermint to use storage at height but the header will be removed in deeper layer

File: proto/ethermint/evm/v1alpha1/query.proto [hunk 2]
```diff
@@ -216,7 +216,10 @@ message QueryBlockLogsResponse {
 
 // QueryBlockBloomRequest is the request type for the Query/BlockBloom RPC
 // method.
-message QueryBlockBloomRequest {}
+message QueryBlockBloomRequest {
+  // height of the block which we want to query the bloom filter
+  int64 height = 1;
```
Comment (fedekunze) [inline proto/ethermint/evm/v1alpha1/query.proto:221]: can you add the note here? otherwise someone might revert the change by accident

Comment (thomas-nguy) [reply proto/ethermint/evm/v1alpha1/query.proto:224]: sure, added more comments and changelog

**Review by fedekunze — APPROVED** [review]

**Comment by fedekunze** [comment]
@thomas-nguy can you fix the conflicts?
